lib/simplifyExpression/basicsSearch/removeAdditionOfZero.js

```diff
@@ -1,30 +1,32 @@
-const clone = require('../../util/clone');
+const {canApplyRule, applyRule} = require('math-rules');
 
 const ChangeTypes = require('../../ChangeTypes');
+const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
+// e.g. 2 + 0 -> 2
+const REMOVE_ADDING_ZERO = defineRuleString('#a + 0', '#a');
+
+// e.g. 0 + 2 -> 2
+const REMOVE_ADDING_ZERO_REVERSE = defineRuleString('0 + #a', '#a');
+
 // If `node` is an addition node with 0 as one of its operands,
 // remove 0 from the operands list. Returns a Node.Status object.
 function removeAdditionOfZero(node) {
-  if (node.op !== '+') {
-    return Node.Status.noChange(node);
+  // TODO(kevinb): merge these two rules together
```

Comment:
should we keep these todos here or do we already have this in `math-rules` for reference?